tested/judge/execution.py

```diff
@@ -96,6 +96,19 @@ def execute_file(
     return result
 
 
+def copy_workdir_files(bundle: Bundle, context_dir: Path):
+    for file in os.listdir(bundle.config.workdir):
+        origin = bundle.config.workdir / file
+        _logger.debug("Copying %s to %s", origin, context_dir)
+        if os.path.isfile(origin):
+            shutil.copy2(origin, context_dir)
+        elif os.path.isdir(origin) and (
+                not str(file).startswith("context") and
```

Comment:
I would use the `context_prefix` function from the language config if possible, so we don't have to hardcode "context" here.